pkgs/applications/virtualization/gvisor/default.nix

```diff
@@ -9,19 +9,26 @@
 
 buildGoModule rec {
   pname = "gvisor";
-  version = "20220905.0";
+  version = "20220919.0";
+
+  # gvisor provides a synthetic go branch (https://github.com/google/gvisor/tree/go)
```

Comment:
I'm assuming that only `Merge release-** (automated)` commits should be used? I'd add that here as well.